release notes/v0.5.0.md

```diff
@@ -0,0 +1,103 @@
+xk6-browser v0.5.0 is here! :tada:
+
+This minor release contains a new feature, a few important stability fixes, some breaking changes, improvements, and a continuation of our efforts to migrate to async APIs. Coinciding with this release, are the up-to-date documentation, which can be found on [https://k6.io/docs/javascript-api/xk6-browser/](https://k6.io/docs/javascript-api/xk6-browser/).
```

Comment:
Should we clarify that it's not done yet?

```suggestion
This minor release contains a new feature, a few important stability fixes, some breaking changes, improvements, and a continuation of our efforts to migrate to async APIs. Coinciding with this release, is the up-to-date documentation, which can be found at [https://k6.io/docs/javascript-api/xk6-browser/](https://k6.io/docs/javascript-api/xk6-browser/). Only a few of the most used classes like `Browser` and `BrowserContext` are currently well documented, and we're working hard on updating the rest of the API.
```

Reply:
I wasn't sure whether to do that, but it does help clarify things for users if they come across some docs that aren't fully fleshed out vs others which are.